frontend/app/forms/MemberForm.scala

```diff
@@ -47,6 +53,25 @@ object MemberForm {
 
   case class FeedbackForm(category: String, page: String, feedback: String, name: String, email: String)
 
+
+  implicit val productFeaturesFormatter: Formatter[FeatureChoice] = new Formatter[FeatureChoice] {
```

Comment:
Unfortunately it isn't. If I do not set it, when I inspect the type in Intellij I end up with a one meter long string. Blame it on the compiler...